proxy/coco/src/peer.rs

```diff
@@ -261,6 +274,24 @@ impl Api {
         Ok(project_meta)
     }
 
+    /// Query the network for providers of the given [`RadUrn`].
+    ///
+    /// Notes:
+    ///   * there is no guarantee that a peer who claims to provide the [`RadUrn`] actually has it,
+    ///     nor that it is reachable using any of the addresses contained in [`PeerInfo`].
+    ///
+    ///   * there is no guarantee that any peer ever replies to the query, meaning that awaiting for
+    ///     a reply might never terminate, requiring callers of this function to timeout on
+    ///     `next()`.
+    #[warn(clippy::future_not_send)]
```

Comment:
This is a red flag, if we see this warning that means we need to refactor until it is `Send`. If not when used/applied anywhere it will lead to compilation errors.

Reply:
This is my bad, I had set it to see if there would be any further warnings just as an experiment and I forgot to drop it. Done now. Sorry!